airgun/entities/host_new.py

```diff
@@ -300,6 +300,30 @@ def get_errata_by_type(self, entity_name, type):
         view.content.errata.table.wait_displayed()
         return view.read(widget_names="content.errata.table")
 
+    def get_errata_type_counts(self, entity_name):
+        """
+        Get errata counts for each type of errata on selected host.
+
+        Args:
+            entity_name: Name of the host.
+        Returns:
+            errata_counts (dict): Dictionary with counts of each type of errata.
+        """
+
+        errata_types = ['Security', 'Bugfix', 'Enhancement']
+        errata_counts = {type: 0 for type in errata_types}
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.wait_displayed()
```

Comment:
- Can we use `waif_for_element()` method (or `wait_for()`) just to avoid use of `time.sleep(sec)` [here test module](https://github.com/SatelliteQE/robottelo/pull/14773/files#diff-bd6863972772cca482646a04c83fba2589bdb45b1fcfdb0c4d544cd9fd17c920R1324), 
- Can take reference from same file.
```suggestion
        view.wait_displayed()
        self.browser.wait_for_element(<locator>, exception=False)
        OR
        wait_for(lambda: view.<web_element difined in VIEW>.is_displayed, timeout=5)
```

Reply:
@vijaysawant I dont think it is valid to hold this PR for the robottelo change. He is using the wait_disaplyed() internally uses the wait_for.

Reply:
- No no @omkarkhatavkar, I am not holding Ack on PR, as in robottelo PR we are refreshing browser and immediatly calling this function, so I am thinking what if we use `wait_for_element` method.
- As in `wait_displayed()` we could pass timeout explicitly, may be this could help
`view.wait_displayed(timeout=5, delay=2)` @LadislavVasina1

Reply:
But I need to wait before calling this function and mainly before the refresh in the test.

Reply:
as suggested changes doesn't work (I saw your response in rebottelo PR), I will leave upto you how to use `wait_for` / `wait_displayed` in this airgun PR.